casts/free-podcasts-screencasts-en.md

```diff
@@ -181,6 +181,7 @@
 * [HTTP203](https://developers.google.com/web/shows/http203/podcast) - Surma, Jake (podcast)
 * [JavaScript Air](https://javascriptair.com) - Kent C. Dodds, Dan Abramov, Matt Zabriskie, Pam Selle, Lin Clark, Brian Lonsdorf, Iheanyi Ekechukwu, Tyler McGinnis, Kyle Simpson, et al. (podcast)
 * [JavaScript Jabber](https://javascriptjabber.com) - Aimee Knight, AJ O'Neal, Charles Wood, Dan Shappir, Steve Edwards, et al. (podcast)
+* [Javascript tutorials for beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7rrvgG7MLNIMSTzVCDZZcT4) - Telusko (podcast)
```

Comment:
It is a video tutorial. Please move it to courses file

Reply:
I have done the necessary changes. Please have a look.